PiTool/pitool_en.ts

```diff
@@ -263,7 +263,7 @@
     </message>
 </context>
 <context>
-    <name>ConnectModeSwicth</name>
+    <name>ConnectModeSwitch</name>
```

Comment:
This label may need to correlate with all the other mentions of it, so better change them all (including renaming files) or leave it alone?

Reply:
You're quite right, I'll revert that.